### ISSUE TITLE
Can't add comments to projects or tasks

### ISSUE THREAD

**Issue by b-trav** [AllediaWordPress/UpStream#597]
## Expected Behavior

Add comments to tasks in either front-end or back-end

## Current Behavior

It is not currently possible to add comments to task in either front-end or back-end

## Steps to Reproduce (for bugs)

1. Try to add a comment to a project or task
## Context

The user's profile should have editor disabled for being able to reproduce this.

## Your Environment

* Version used: 1.20.1
* various
* various
* private

**Comment by stevejburge** [comment]
Thanks for reporting this. We have a beta package ready with a fix: http://packages-tmp.s3.amazonaws.com/b93cda42f8402eea98671f42740aa120/upstream-1.20.2-beta.4.zip

**Comment by b-trav** [comment]
I am attempting to use the latest version of the plugin from the development branch, but when I attempt to activate the plugin I get the following error:
```
"PHP message: PHP Fatal error:  Class 'Pimple\Container' not found in /------/UpStream/includes/class-up-container.php on line 3"
```
At the moment, I have no working plugin at all. Please tell me how I can get the plugin working again.

**Comment by stevejburge** [comment]
Hi @b-trav Could email our support help@upstreamplugin.com with more details on this, please? We can't do support via Github.

**Comment by b-trav** [comment]
I understand you can't do support via Github, and I have already reported this via support on help@upstreamplugin.com, but how is this NOT a bug?

**Comment by stevejburge** [comment]
Hi @b-trav Yes, Github is just for code discussions, please

Yes, the comments issue is a bug. We have a beta package ready with a fix: http://packages-tmp.s3.amazonaws.com/b93cda42f8402eea98671f42740aa120/upstream-1.20.2-beta.4.zip

**Comment by b-trav** [comment]
This issue occurs if a user has checked "Disable the visual editor when writing" in their user profile.

This issue is not fixed as of version 1.21.0

**Comment by andergmartins** [comment]
If you still have any issue related this, we found a new place with a similar issue, preventing users who disabled editor to edit projects in the front-end. Here is a new beta. We will release next week:

http://packages-tmp.s3.amazonaws.com/b2e215e894596bb3ced09a896330d312/upstream-frontend-edit-1.7.7-beta.1.zip
